README-ru.md

```diff
@@ -0,0 +1,551 @@
+
+# DashKit
+
+Библиотека для рендеринга сетки дашборда.
+
+## Установка
+
+```bash
+npm i @gravity-ui/dashkit @gravity-ui/uikit
+```
+
+## Описание
+
+Библиотека позволяет выстраивать виджеты в сетку, изменять их размеры, добавлять новые и удалять их.
+Виджет представляет из себя react-компонент – к примеру, текст, график, картинка и т.п.
+
+Добавление новых виджетов осуществляется через систему плагинов.
+
+### Плагины
+
+Плагины нужны для создания собственных виджетов.
+
+### Свойства
+
+```ts
+type ItemManipulationCallback = (eventData: {
+    layout: Layouts;
+    oldItem: Layout;
+    newItem: Layout;
+    placeholder: Layout;
+    e: MouseEvent;
+    element: HTMLElement;
+}) => void;
+
+interface DashKitProps {
+  config: Config;
+  editMode: boolean;
+  onItemEdit: ({id}: {id: string}) => void;
+  onChange: (data: {config: Config; itemsStateAndParams: ItemsStateAndParams}) => void;
+  onDrop: (dropProps: ItemDropProps) => void;
+  onItemMountChange: (item: ConfigItem, state: {isAsync: boolead; isMounted: boolean}) => void;
+  onItemRender: (item: ConfigItem) => void;
+
+  onDragStart?: ItemManipulationCallback;
+  onDrag?: ItemManipulationCallback;
+  onDragStop?: ItemManipulationCallback;
+  onResizeStart?: ItemManipulationCallback;
+  onResize?: ItemManipulationCallback;
+  onResizeStop?: ItemManipulationCallback;
+
+  defaultGlobalParams: GlobalParams;
+  globalParams: GlobalParams;
+  itemsStateAndParams: ItemsStateAndParams;
+  settings: SettingsProps;
+  context: ContextProps;
+  overlayControls?: Record<string, OverlayControlItem[]> | null;
+  overlayMenuItems?: MenuItems[] | null;
+  noOverlay?: boolean;
+  focusable?: boolean;
+  draggableHandleClassName?: string;
+  getPreparedCopyItemOptions?: (options: PreparedCopyItemOptions) => PreparedCopyItemOptions;
+  onCopyFulfill?: (error: null | Error, data?: PreparedCopyItemOptions) => void;
+}
+```
+
+- `config` — [кофигурация](#Config).
+- `editMode`— включает или отключает режим редактирования.
+- `onItemEdit` – вызывается по клику для редактирования виджета.
+- `onChange` — вызывается при изменении значения `config` или [`itemsStateAndParams`](#itemsStateAndParams).
+- `onDrop` — вызывается, когда элемент перетаскивается из `ActionPanel` (панели действий) с помощью `(#DashKitDnDWrapper)`.
+- `onItemMountChange` — вызывается при изменении состояния монтирования элемента.
+- `onItemRender` — вызывается по завершении рендеринга элемента.
+- `defaultGlobalParams`, `globalParams` — [параметры](#Params), которые влияют на все виджеты. В DataLens `defaultGlobalParams` — глобальные параметры, заданные в настройках дашборда, а `globalParams` — глобальные параметры, которые можно задать в URL.
+- `itemsStateAndParams` — [itemsStateAndParams](#itemsStateAndParams).
+- `settings` — настройки `DashKit`.
+- `context`: объект, который пробросится в виде свойства во все виджеты.
```

Comment:
May be use one common style for all props? `-` or `:` for all options?
And, by the way, why did you use `-` instead of `:`? There is `:` in en readme

Reply:
thx, fixed